django_tiptap/widgets.py

```diff
@@ -29,4 +28,32 @@ def set_config(config: dict = None) -> dict:
     def get_context(self, *args, **kwargs) -> Dict[str, Any]:
         context = super().get_context(*args, **kwargs)
         context["widget"]["config"] = self.config
+
+        if (context["widget"]["config"].get("tooltips")):
```

Comment:
In contrast to JS, it is more idiomatic in Python to not wrap the statement your evaluating in parentheses.  
Also, if you just want to check whether or not a given key is contained in a dict, without the need of assigning the corresponding value to a variable, you can use the following construct: `if key in dict:`

Putting it both together could look something like this:
```suggestion
        if "tooltips" in context["widget"]["config"]:
```

Reply:
Cool Stuff ⚡